experimental/FTheoryTools/src/LiteratureModels/constructors.jl

```diff
@@ -172,23 +161,26 @@ function literature_model(model_dict::Dict{String, Any}; model_parameters::Dict{
     
     # Currently, support only for toric bases
     @req base_space isa NormalToricVariety "Construction of literature models over concrete bases currently limited to toric bases"
-    for (key, value) in model_sections
-      @req value isa ToricDivisor "Construction of literature models over concrete bases currently requires toric divisors as model sections"
-    end
 
-    # Are all model sections specified?
-    @req all(k->haskey(model_sections, k), model_dict["model_data"]["divisors_user_must_specify"]) "Not all model sections are specified"
+    # FIXME: Append model_sections to defining_classes. This might need fixing/extending in the future...
+    defining_classes_provided = merge(model_sections, defining_classes)
+    for (key, value) in defining_classes_provided
+      if (value isa ToricDivisor || value isa ToricDivisorClass || value isa ToricLineBundle) == false
+        error("Construction of literature models over concrete bases currently requires defining classes (and model sections) to be provided as toric divisor (classes) or line bundles")
```

Comment:
So which "model sections" is this referring to now? I think it's referring to things like `w` in the SU(5)xU(1) model, which we used to refer to as "model sections" but are now called "defining classes", which you also mention here. Is this phrasing used for backward compatibility? I don't want to confuse people with our new versus old definitions of "model sections". Here for instance, you say the model sections must be given as divisors, divisor classes, or line bundles; but our new use of "model sections" refers to the sections themselves. We should discuss.